Hedvig/Assets/HedvigColors.swift

```diff
@@ -18,13 +18,13 @@ extension UIColor {
     static let black = UIColor.black
     static let turquoise = UIColor(red: 0.11, green: 0.91, blue: 0.71, alpha: 1.0)
     static let purple = UIColor(red: 0.40, green: 0.12, blue: 1.00, alpha: 1.0)
-    static let darkPurple = UIColor(red: 0.03, green: 0.02, blue: 0.27, alpha: 1.0)
-    static let blackPurple = UIColor(red: 0.06, green: 0.00, blue: 0.48, alpha: 1.0)
+    static let blackPurple = UIColor(red: 0.03, green: 0.02, blue: 0.27, alpha: 1.0)
+    static let darkPurple = UIColor(red: 0.06, green: 0.00, blue: 0.48, alpha: 1.0)
     static let darkGray = UIColor(red: 0.61, green: 0.61, blue: 0.67, alpha: 1.0)
     static let lightGray = UIColor(red: 0.91, green: 0.93, blue: 0.94, alpha: 1.0)
     static let offBlack = UIColor(red: 0.25, green: 0.25, blue: 0.31, alpha: 1.0)
     static let offWhite = UIColor(red: 0.98, green: 0.98, blue: 0.99, alpha: 1.0)
-    static let green = UIColor(red: 0.11, green: 0.91, blue: 0.71, alpha: 1.0)
+    static let green = UIColor(red: 0, green: 0.57, blue: 0.46, alpha: 1.0)
```

Comment:
was this not correct before? 🤔

Reply:
@Iamsamwhoami Green was the same value as turquoise. Changed it due to a new green color specified in the design. It's quite dark though (#009175), so maybe it should be named darkGreen instead?

Reply:
@isaacSennerholt https://github.com/HedvigInsurance/brand/blob/master/colors.ts#L14 green is `0.11, 091, 0.71` 🤔 ask Zak about what we should call this new one.

Reply:
@Iamsamwhoami But that doesn't make sense though, as the green you're referring to in brand repo also is same value as turquoise. Why is green and turquoise the same to begin with? Can this new color, which is more green-like than turquoise, be green for now? If not, we ought to review the old green color and try to change it's use to turquoise.

Reply:
@isaacSennerholt it doesn't make sense, but turquoise isn't really used (only once in the old app) and "#1BE9B6" isn't turquoise 😅it's green. So I guess this new one should be called darkGreen?  🤷‍♂️

Reply:
@isaacSennerholt Hmm, I would say remove green, call this new one darkGreen.